### PR TITLE
Challenge 1

### PR DESCRIPTION
**Pull request by Eyansky**
#### What's this PR do?
WeConnect provides a platform that brings businesses and individuals together. This platform creates awareness for businesses and gives the users the ability to write reviews about the businesses they have interacted with.  

#### Where should the reviewer start?
README file

#### How should this be manually tested?
-git clone https://github.com/Eyansky/WeConnect_App.git
-cd WeConnect_App
- open login.html with browser

#### Any background context you want to provide?
github page - https://eyansky.github.io/WeConnect.github.io/register.html

#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
login
registration
screen shot 2018-03-01 at 15 02 54
screen shot 2018-03-01 at 15 03 20
screen shot 2018-03-01 at 15 03 41
screen shot 2018-03-01 at 15 04 14
screen shot 2018-03-01 at 15 04 35
screen shot 2018-03-01 at 15 05 04
screen shot 2018-03-01 at 15 05 24

#### Questions:
n/a

### CODE REVIEW

File: Designs/UI/SearchLocation.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    </div>
+        </div>
+
+        <div id="footer">
+           <p> copywright &copy; 2018: Author Ian Mwangi</p>
+        </div>
+    </div>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+</body>
+</html>
```
Comment (jmnyarega) [inline Designs/UI/SearchLocation.html:48]: Research on how to indent your HTML code, It should look much cleaner

File: Designs/UI/businessRegistration.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+              </form>
+        </div>
+
+        <div id="footer">
+           <p> copywright &copy; 2018: Author Ian Mwangi</p>
+        </div>
+    </div>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+</body>
+</html>
```
Comment (jmnyarega) [inline Designs/UI/businessRegistration.html:89]: Add **ONE** extra line on each of your files

File: Designs/UI/catalog.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+        <div class="container">
+                <div class="card-deck mb-3 text-center">
+                  <div class="card mb-4 box-shadow">
+                    <div class="card-header">
+                      <h4 class="my-0 font-weight-normal">Business 1</h4>
+                    </div>
+                    <div class="card-body">
+                        <div class="rating">
+                            <span>☆</span><span>☆</span><span>☆</span><span>☆</span><span>☆</span>
```
Comment (jmnyarega) [inline Designs/UI/catalog.html]: I don't understand why you require all those spans, cant you just use one span?

**Comment by clemwek** [comment]
Good work, you can add a screenshot to your PR message to show how it looks at a glance.

File: Designs/UI/SearchLocation.html (hunk 1 above)
Comment (Eyansky) [reply Designs/UI/SearchLocation.html:48]: working on it

**Pull request merged**